### DISCUSSION TITLE
Does the openssl 3.x implement the SP 800-90Ar1 section 11.3.4?

### DISCUSSION THREAD

**Discussion by laogaolao** [openssl/openssl#22655] [Q&A] [answered]
hi Guys,
The SP 800-90Ar1 11.3.4 is about reseed test, does openssl 3.x implemented it? I see there is test_seed in the test folder, does it meet the requirements of section 11.3.4? Thanks.

**Comment by paulidale** [comment, marked as answer]
I believe so but questions about FIPS compliance **must** be directed to an accredited FIPS lab.  They are the only canonical reference body outside of NIST for this kind of question.
